### PR TITLE
remove deprecated const EOL = PHP_EOL; in AbstractHtmlElement

### PR DESCRIPTION
**Pull request by samsonasik**
in favor of https://github.com/zendframework/zf2/pull/7417

### CODE REVIEW

**Comment by samsonasik** [comment]
merge-able ?

**Comment by samsonasik** [comment]
@weierophinney merge-able ?

**Comment by samsonasik** [comment]
@weierophinney any chance to merge this?

File: src/Helper/AbstractHtmlElement.php [hunk 1]
```diff
@@ -12,13 +12,6 @@
 abstract class AbstractHtmlElement extends AbstractHelper
 {
     /**
-     * EOL character
-     *
-     * @deprecated just use PHP_EOL
-     */
-    const EOL = PHP_EOL;
```
Comment (weierophinney) [inline src/Helper/AbstractHtmlElement.php]: We cannot remove it until a new major version is released.

That said, we can remove usage of it within the tests. When I merge, I'll reinstate the constant.

**Pull request merged**